soccer/gameplay/evaluation/touchpass_positioning.py

```diff
@@ -0,0 +1,100 @@
+import robocup
+import constants
+import main
+import evaluation.passing
+
+## The Touchpass Positioner finds the best location within a rectangle
```

Comment:
Maybe make this just a bit more descriptive by mentioning that it's used for finding positions for the receiver in the touchpass tactic.  It also wouldn't hurt to give a quick description of what you mean by "best" location here.